awx/main/models/inventory.py

```diff
@@ -1526,12 +1533,17 @@ class rhv(PluginFileInjector):
     initial_version = '2.9'
     namespace = 'ovirt'
     collection = 'ovirt'
+    downstream_namespace = 'redhat'
+    downstream_collection = 'rhv'
 
 
 class satellite6(PluginFileInjector):
     plugin_name = 'foreman'
     namespace = 'theforeman'
     collection = 'foreman'
+    downstream_namespace = 'redhat'
+    downstream_collection = 'satellite'
+    use_fqcn = True
```

Comment:
I don't love that I had to add this new property for this _one_ case where the upstream required the FQCN.... but that's what it does.

https://github.com/theforeman/foreman-ansible-modules/blob/develop/plugins/inventory/foreman.py#L28